lib/modules/datasource/crate/index.spec.ts

```diff
@@ -70,6 +70,62 @@ function setupErrorGitMock(): {
   return { mockClone };
 }
 
+function setupErrorShallowGitMock(): {
```

Comment:
please inline, those functions are only used once. makes no sense to have them separate